internal/util-logging/src/main/scala/sbt/util/InterfaceUtil.scala

```diff
@@ -43,6 +47,18 @@ object InterfaceUtil {
       case None    => Optional.empty[A]()
     }
 
+  def l2jl[A](l: List[A]): ju.List[A] = {
```

Comment:
I just followed how you did the jo2o up above because I'm assuming you don't want collection converters brought in?

Reply:
Sounds good. Not bringjng in dependency is a good default stance for bottom layers like this.